packages/StaticTypeMapper/PhpDocParser/IdentifierTypeMapper.php

```diff
@@ -53,6 +53,11 @@ public function getNodeType(): string
      * @param IdentifierTypeNode $typeNode
      */
     public function mapToPHPStanType(TypeNode $typeNode, Node $node, NameScope $nameScope): Type
+    {
+        return $this->mapIdentifierTypeNode($typeNode, $node);
+    }
+
+    public function mapIdentifierTypeNode(IdentifierTypeNode $typeNode, Node $node): Type
```

Comment:
I think new `mapIdentifierTypeNode()` method can be reverted, use back `mapToPHPStanType()` seems ok

Reply:
Oh, I see, you don't want `NameScope` to be involved here, so it seems ok then 👍